code/online_challenges/src/hackerrank/jumping_on_the_clouds/jumping_on_the_clouds.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+
+int main()
+{
+    int n, i, jump = 0;
+    scanf ("%d", &n);
+    int c[n];
+    for (i=0;i<n;i++)
+    {
+        scanf ("%d", &c[i]);
+    }
+    for (i=0;i<n;i++)
+    {
+        if (i+2 < n && !c[i+2])
+        {
+            jump++;
+            i++;
+        }
+        else if (i+1 < n && !c[i+1])
+        {
+            jump++;
+        }
+    }
+    printf ("%d", jump);
```

Comment:
Reformat this file using clang format with 4 spaces.

Reply:
Sir, it has 4 space format. Can you please tell.